index.html

```diff
@@ -42,5 +43,224 @@ <h2 class="Supportingtext Title2">I'm a Software Developer</h2>
+          <ul class="list_feat list_card_gral">
+            <li class="li_btn_2">HTML</li>
+            <li class="li_btn_2">Bootstrap</li>
+            <li class="li_btn_2">Ruby</li>
+          </ul>
+        </div> 
+        <a href="#" class="SeeProjectBtn project_btn_2">See Project</a>
+      </div>
+    </div>
+    <div class="sec_card_cnt work-4">
+      <div class="info_card">
+        <div class="cnt_info_card">
+          <h1 class="Title1_card">Profesional Art</h1>
+          <h2 class="Title2_card">Printing Data</h2>
+          <p class="CardDescrip Simpletext">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard.</p>
+          <ul class="list_feat list_card_gral">
+            <li class="li_btn_2">HTML</li>
+            <li class="li_btn_2">Bootstrap</li>
+            <li class="li_btn_2">Ruby</li>
+          </ul>
+        </div> 
+        <a href="#" class="SeeProjectBtn project_btn_2">See Project</a>
+      </div>
+    </div>
+    <div class="sec_card_cnt work-5">
+      <div class="info_card">
+        <div class="cnt_info_card">
+          <h1 class="Title1_card">Profesional Art</h1>
+          <h2 class="Title2_card">Printing Data</h2>
+          <p class="CardDescrip Simpletext">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard.</p>
+          <ul class="list_feat list_card_gral">
+            <li class="li_btn_2">HTML</li>
+            <li class="li_btn_2">Bootstrap</li>
+            <li class="li_btn_2">Ruby</li>
+          </ul>
+        </div> 
+        <a href="#" class="SeeProjectBtn project_btn_2">See Project</a>
+      </div>
+    </div>
+    <div class="sec_card_cnt work-6">
+      <div class="info_card">
+        <div class="cnt_info_card">
+          <h1 class="Title1_card">Profesional Art</h1>
+          <h2 class="Title2_card">Printing Data</h2>
+          <p class="CardDescrip Simpletext">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard.</p>
+          <ul class="list_feat list_card_gral">
+            <li class="li_btn_2">HTML</li>
+            <li class="li_btn_2">Bootstrap</li>
+            <li class="li_btn_2">Ruby</li>
+          </ul>
+        </div> 
+        <a href="#" class="SeeProjectBtn project_btn_2">See Project</a>
+      </div>
+    </div>
+  </div>
+
+  </section>
+
+  <!-- Section About -->
+  <section class="about back_dark">        
+    <div class="about_container">
+      <h1 class="Tittle_about">About me</h1>
+      <p class="DescripAbout Simpletext">Hello I'm a software developer! I can help you build a product, feauture or website. Look through some of my work and expererience!. If you like what you see and have a prject you need coded, don't hesitate to contact me.</p>
```

Comment:
As with my previous comment, I realize that you have a font family of ```poppins``` on this text whereas it is ```Inter``` in the figma design. You could change this as well to make your project identical to the figma design 👍

Reply:
The focus of my comment here as well is on the p tag on line 205 👍

Reply:
Done. Thank You @amon-cofie